src/components/ImageCard/ImageHotspots.test.jsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { act, isDOMComponent } from 'react-dom/test-utils';
+import { render } from '@testing-library/react';
+import { screen } from '@testing-library/dom';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
is this necessary to bring in for the expect assertions we have here?

Reply:
https://github.com/IBM/carbon-addons-iot-react/issues/1400